prime-router/src/main/kotlin/fhirengine/utils/FHIRBundleHelpers.kt

```diff
@@ -80,7 +80,8 @@ object FHIRBundleHelpers {
         val provenanceResource = try {
             fhirBundle.entry.first { it.resource.resourceType.name == "Provenance" }.resource as Provenance
         } catch (e: NoSuchElementException) {
-            throw IllegalStateException("The FHIR bundle does not contain a Provenance resource")
+            fhirBundle.addEntry().resource = Provenance()
+            fhirBundle.entry.first { it.resource.resourceType.name == "Provenance" }.resource as Provenance
```

Comment:
Does this new behavior need to be tested in unit tests? Or is that not necessary since this is just for connectathon?